core/data_migration_scripts/4_1_0_stages/EE_DMS_4_1_0_checkins.dmsstage.php

```diff
@@ -97,7 +112,7 @@ private function _try_to_find_datetime($old_attendee)
 
         $new_event_id = $this->get_migration_script()->get_mapping_new_pk($wpdb->prefix."events_detail", $old_attendee['event_id'], $wpdb->posts);
         if (! $new_event_id) {
-            $this->add_error(sprintf(__("Could nto find new event ID with old event id '%d', on attendee row %s; and because of that couldnt find the correct datetime for Check-in", "event_espresso"), $old_attendee['event_id'], $this->_json_encode($old_attendee)));
+            $this->add_error(sprintf(esc_html__('Could not find new event ID with old event ID %d, on attendee row %s; and because of that couldn\'t find the correct datetime for Check-in', 'event_espresso'), $old_attendee['event_id'], $this->_json_encode($old_attendee)));
```

Comment:
since this translation string will break due to the typo fix anyways, let's also switch it over to using numbered placeholders (and format it), ie: 

```php
$this->add_error(
    sprintf(
        esc_html__(
            'Could not find new event ID with old event ID %1$d, on attendee row %2$s; and because of that couldn\'t find the correct datetime for Check-in', 
            'event_espresso'
        ),
        $old_attendee['event_id'],
        $this->_json_encode($old_attendee)
    )
);
```